lms/djangoapps/certificates/generation_handler.py

```diff
@@ -137,9 +208,16 @@ def is_using_certificate_allowlist(course_key):
     return CERTIFICATES_USE_ALLOWLIST.is_enabled(course_key)
 
 
+def _is_using_v2_course_certificates(course_key):
+    """
+    Return True if the course run is using v2 course certificates
+    """
+    return CERTIFICATES_USE_UPDATED.is_enabled(course_key)
+
+
 def _is_on_certificate_allowlist(user, course_key):
     """
-    Check if the user is on the allowlist for this course run
+    Check if the user is on the allowlist, and is enabled for the allowlist, for this course run
```

Comment:
This comment addition threw me off. Is a **user** _enabled_ for the allowlist separate from being _on_ it? Or does the _enabling_ here refer to whether or not allowlisting is enabled for the course run?

Reply:
See _whitelist=True_ in the method content. 

A user can be present in the allowlist model for a given course run, but not have their whitelisting enabled. In other words, they might have previously been allowlisted but we've currently disabled their allowlisting for this course run.